website/docs/docs/dbt-versions/core-upgrade/01-upgrading-to-v1.8.md

```diff
@@ -17,7 +17,7 @@ dbt Labs is committed to providing backward compatibility for all versions 1.x,
 
 ## Keep on latest version 
 
-With dbt Cloud, you can get early access to many new features and functionality before they're in the Generally Available (GA) release of dbt Core v1.8 without the need to manage version upgrades. Refer to the [Keep on latest version](/docs/dbt-versions/upgrade-dbt-version-in-cloud#keep-on-latest-version) setting for more details.
+dbt Cloud is going "versionless." Going forward, you get early access to new features and functionality before they're available in final releases of dbt Core. Select ["Keep on latest version"](/docs/dbt-versions/upgrade-dbt-version-in-cloud#keep-on-latest-version) in your development and deployment environments to get access to everything in dbt Core v1.8 and more.
```

Comment:
```suggestion
dbt Cloud is going "versionless." This means you'll automatically get early access to new features and functionality before they're available in final releases of dbt Core. 

Select ["Keep on latest version"](/docs/dbt-versions/upgrade-dbt-version-in-cloud#keep-on-latest-version) in your development, staging, or deployment environments to get access to everything in dbt Core v1.8 and more.
```

Reply:
Good call @mirnawong1. I think this sentence should either read:
- "development and deployment environments"
- "development, staging, and production environments"

Given that the environment hierarchy is:
- type: development
- type: deployment
    - deployment_type: production
    - deployment_type: staging
    
That is, "production" and "staging" are subtypes of "deployment" (= for running jobs that deploy models on a schedule/trigger) as distinct from "development" (= operating on models interactively in response to user edits/commands)

Reply:
sound good! i'll keep it as development, staging, and production environments to enhance the 'staging' aspect if that's cool!